src/appl_grid.cxx

```diff
@@ -398,10 +398,11 @@ appl::grid::grid(const std::string& filename, const std::string& dirname)  :
 
     if ( ckm2flat ) { 
       if ( ckm2flat->GetNrows()>0 ) { 
-	_ckm2 = std::vector<std::vector<double> >(14, std::vector<double>(14) );
+	_ckm2 = std::vector<std::vector<double> >(14, std::vector<double>(14,0) );
       
-	for ( int ic=0 ; ic<14 ; ic++ ) { 
-	  for ( int id=0 ; id<14 ; id++ ) _ckm2[ic][id] = (*ckm2flat)(ic*14+id); 
+  // Note in normal APPLgrids only 13 partons are put in the flat CKM array
+	for ( int ic=0 ; ic<13 ; ic++ ) { 
+	  for ( int id=0 ; id<13 ; id++ ) _ckm2[ic][id] = (*ckm2flat)(ic*13+id); 
```

Comment:
ok, this looks fine, but I have to check if we can build the applgrids without issues.